convert2rhel/unit_tests/actions/dbus_is_running_test.py

```diff
@@ -0,0 +1,72 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import pytest
+import six
+
+from convert2rhel import actions, unit_tests
+from convert2rhel.actions import dbus_is_running
+from convert2rhel.systeminfo import system_info
+
+
+@pytest.fixture
+def dbus_is_running_action():
+    return dbus_is_running.DbusIsRunning()
+
+
+@pytest.mark.parametrize(
+    ("no_rhsm", "dbus_running", "log_msg"),
+    (
+        (True, True, "Skipping the check because we have been asked not to subscribe this system to RHSM."),
+        (True, False, "Skipping the check because we have been asked not to subscribe this system to RHSM."),
+        (False, True, "DBus Daemon is running"),
+    ),
+)
+def test_check_dbus_is_running(
+    caplog, monkeypatch, global_tool_opts, global_system_info, no_rhsm, dbus_running, log_msg, dbus_is_running_action
+):
+    monkeypatch.setattr(actions.dbus_is_running, "tool_opts", global_tool_opts)
+    global_tool_opts.no_rhsm = no_rhsm
+    monkeypatch.setattr(actions.dbus_is_running, "system_info", global_system_info)
+    global_system_info.dbus_running = dbus_running
+
+    assert dbus_is_running_action.run() is None
+    assert caplog.records[-1].message == log_msg
+
+    assert log_msg == caplog.records[-1].message
+
+
+def test_check_dbus_is_running_not_running(
+    caplog, monkeypatch, global_tool_opts, global_system_info, dbus_is_running_action
+):
+    monkeypatch.setattr(actions.dbus_is_running, "tool_opts", global_tool_opts)
+    global_tool_opts.no_rhsm = False
+    monkeypatch.setattr(actions.dbus_is_running, "system_info", global_system_info)
+    global_system_info.dbus_running = False
+
+    dbus_is_running_action.run()
+
+    unit_tests.assert_actions_result(
+        dbus_is_running_action,
+        status="ERROR",
+        error_id="DBUS_DAEMON_NOT_FOUND",
```

Comment:
That would need to be changed if https://github.com/oamg/convert2rhel/pull/779/files#r1146656910 is accepted 

```suggestion
        error_id="DBUS_DAEMON_NOT_RUNNING",
```